benchmarks/templates/benchmarks/competition.html

```diff
@@ -207,23 +239,52 @@ <h4>Does my model have to beat the current most brain-like model on brain-score.
                     Nope! You only have to be the best out of all the competition submissions.
                 </p>
             </li>
+            <li>
+                <h4>Is public data available?</h4>
+                <p>
+                    Yes. Public data splits of many of the benchmarks can be found
+                    <a href="https://github.com/brain-score/brain-score/blob/master/brainscore/benchmarks/public_benchmarks.py#L87">here</a>.
+                    Use the list_public_assemblies function
+                    <a href="https://github.com/brain-score/brain-score/blob/master/brainscore/benchmarks/public_benchmarks.py#L87">
+                        here</a> as well to view all available public data.
+                </p>
+            </li>
         </ol>
     </section>
 
     {# sponsors #}
     <section class="section container">
-        <h2 class="title is-2">Sponsors</h2>
-        <a href="https://quest.mit.edu/">
-            <img src="{% static "/benchmarks/img/competition_logos/quest.PNG" %}" alt="MIT Quest"
-                 width="200" height="100">
-        </a>
-        <a href="https://engineering.purdue.edu/C-BRIC">
-            <img src="{% static "/benchmarks/img/competition_logos/cbric.png" %}" alt="C-BRIC"
-                 width="200" height="100">
-        </a>
-        <a href="https://mitibmwatsonailab.mit.edu/">
-            <img src="{% static "/benchmarks/img/competition_logos/ibm.jpg" %}" alt="MIT-IBM Watson AI Lab"
-                 width="200" height="100">
-        </a>
+        <div class="columns">
+            <h2 class="title is-2">Sponsors</h2>
+            <div class="column">
+                <br><br><br><br><br>
```

Comment:
this doesn't look right and will probably lead to a ton of useless whitespace on mobile --> use bulma!